index.js

```diff
@@ -44,7 +44,7 @@ types[VISA] = {
   prefixPattern: /^4$/,
   exactPattern: /^4\d*$/,
   gaps: [4, 8, 12],
```

Comment:
Tried googling and found https://technologypartner.visa.com/download.aspx?id=32
-> page 177
`PAN: 4000 0012 3456 2345 678 (19 digits)`
Not definitive, but it's one idea